pywhat/Data/regex.json

```diff
@@ -125,6 +125,102 @@
          "YouTube Video"
       ]
    },
+   {
+      "Name": "Twitter Status",
+      "Regex": "^(https?:\\/\\/(?:twitter\\.com\\/)(?:#!\\/)?(\\w{1,15})\\/status(?:es)?\\/(\\d+))$",

```

Comment:
It may be a good idea to make all of these regexes case insensitive (add `(?i)` after `^`), so they match to uppercase URLs too: HTTPS://TWITTER.COM/BEE_SEC_SAN/STATUS/1413943153732816902?S=21

Reply:
Okay, although the existing YouTube Video [URL] regex doesn't have `(?i)`

Reply:
Make one URL uppercase and test it. If it works, you can make it case insensitive. Also please test other URLs too as these contain IDs and just to be sure that they also work when fully uppercase :D

Reply:
Added `(?:)` to regexes and more tests.